pokeapp/src/components/Sidebar.js

```diff
@@ -55,6 +55,29 @@ const SideBar = ({
     getPokenames(allPokemon);
   }, [allPokemon]);
 
+  const renderOwnedQuantity = () => {
+
+    const selectedPokemonSnapShot = allPokemon.find((pokemon)=> pokemon.id === selectedPokemon.id);
+    if (selectedPokemonSnapShot.quantity > 0){
+      return(
+        <div className="pokeball" >
+        <div className="pokeball-top"></div>
+        <div className="pokeball-bottom"></div>
+        <h4
+          className={
+            selectedPokemonSnapShot.quantity >= 100
+              ? "pokeball-font-hundreds"
+              : "pokeball-font"
```

Comment:
Interesting design. Different font if you have more than 100

Reply:
Interesting...